src/services/fluentd.py

```diff
@@ -16,9 +18,6 @@ def __init__(self, docker_network, config_dir):
                 f"{self.config_dir / FLUENT_CONF}:/fluentd/etc/{FLUENT_CONF}"
             ],
             "command": ["/bin/sh", "-c", f"sleep 10 && fluentd -c /fluentd/etc/{FLUENT_CONF}"],
```

Comment:
don't need the `sleep 10`

Reply:
yep, removed

Reply:
Going to revert this, seems to be some magic-fu which makes things work... Not sure why, not sure I have time to work out why...